src/main/java/racinggame/model/Car.java

```diff
@@ -0,0 +1,44 @@
+package racinggame.model;
+
+import com.sun.media.sound.InvalidFormatException;
+
+public class Car {
+
+  public static final String NOT_ALPHABET_CAR_NAME = "car name containing not alphabet";
+  public static final String CAR_NAME_LENGTH_OVER = "car name length is invalid";
+  public String carName;
+  public int moveCount;
+  private static final int CAR_NAME_LENGTH = 5;
+  private final int MOVE_BOUNDARY = 4;
```

Comment:
```suggestion
  public static final String NOT_ALPHABET_CAR_NAME = "car name containing not alphabet";
  public static final String CAR_NAME_LENGTH_OVER = "car name length is invalid";
  private static final int CAR_NAME_LENGTH = 5;
  private final int MOVE_BOUNDARY = 4;
  
  public String carName;
  public int moveCount;
```
static한 변수가 위에 오는 것이 컨벤션 입니다 👍